editor/project_export.cpp

```diff
@@ -541,7 +547,9 @@ void ProjectExportDialog::_delete_preset() {
 void ProjectExportDialog::_delete_preset_confirm() {
 	int idx = presets->get_current();
 	_edit_preset(-1);
+#ifndef ANDROID_ENABLED
 	export_button->set_disabled(true);
+#endif
```

Comment:
There might be less Android specific hacks in the file if you still run the logic that creates those buttons, but `hide()` them on Android maybe?